bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="wundergroundupdatereceiver"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="wundergroundUpdateReceiver">
+		<label>WundergroundUpdateReceiver Binding Thing</label>
+		<description>An endpoint thing that can receive and propagate HTTP GET updates meant for a particular station id at
+			wunderground.com</description>
+
+		<channel-groups>
+			<channel-group id="metadata" typeId="metadata-group"/>
+			<channel-group id="wind" typeId="wind-group"/>
+			<channel-group id="temperature" typeId="temperature-group"/>
+			<channel-group id="humidity" typeId="humidity-group"/>
+			<channel-group id="rain" typeId="rain-group"/>
+			<channel-group id="sunlight" typeId="sunlight-group"/>
+			<channel-group id="pressure" typeId="pressure-group"/>
+			<channel-group id="pollution" typeId="pollution-group"/>
+		</channel-groups>
+
+		<representation-property>stationId</representation-property>
+		<config-description>
+			<parameter name="stationId" type="text" required="true" pattern="\w+">
+				<label>The stationid as configured in the weather station</label>
```

Comment:
This label is too long, try to keep them less than 30 characters. Labels should have each word start in a capital letter.